### PR TITLE
Clarify artifact format usage with spin CLI

### PR DESCRIPTION
**Pull request by ejsuncy**
Because it took me far too long to discover how to use this option correctly.

### CODE REVIEW

File: reference/artifacts/index.md [hunk 1]
```diff
@@ -116,3 +116,24 @@ within pipelines, or produced by pipelines.
   "location": "us-east-1"
 }
 ```
+
+### Example for Providing Artifacts with `spin` CLI
```
Comment (dorbin) [inline reference/artifacts/index.md]:
```suggestion
### Example for providing artifacts with `spin` CLI
```

File: reference/artifacts/index.md [hunk 2]
```diff
@@ -116,3 +116,24 @@ within pipelines, or produced by pipelines.
...
+
+When executing pipelines with the `spin` CLI, the `-t, --artifacts-file` option expects a valid json file containing artifact definitions. For example, you can call `spin pipeline execute --application ${APPLICATION} --name ${PIPELINE} --parameter-file ${PARAMETER_FILE_PATH} --artifacts-file ${ARTIFACTS_FILE_PATH}` with the contents of the file at `${ARTIFACTS_FILE_PATH}` as follows:
```
Comment (dorbin) [inline reference/artifacts/index.md]:
```suggestion
When you execute pipelines with the [`spin` CLI](/guides/spin/), the `-t, --artifacts-file` option expects a valid JSON file containing artifact definitions. For example, you can call...

```
`spin pipeline execute --application ${APPLICATION} --name ${PIPELINE} --parameter-file ${PARAMETER_FILE_PATH} --artifacts-file ${ARTIFACTS_FILE_PATH}`
```

...with the contents of the file at `${ARTIFACTS_FILE_PATH}` as follows:
```

**Review by dorbin — COMMENTED** [review]
Thanks, @ejsuncy. A few editorial comments.

@louisjimenez , fyi and ptal

**Review by louisjimenez — CHANGES_REQUESTED** [review]
Example looks good. Were the editorial changes proposed by @dorbin accidentally overwritten?

**Comment by ejsuncy** [comment]
oh Github...*sigh*. I'll try again

**Comment by ejsuncy** [comment]
OK, that looks better.

**Review by louisjimenez — APPROVED** [review]

**Review by dorbin — APPROVED** [review]

**Pull request merged**